go/enclave/events/subscription_manager.go

```diff
@@ -67,7 +67,7 @@ func (s *SubscriptionManager) AddSubscription(id gethrpc.ID, encryptedSubscripti
 	subscription.Filter.BlockHash = nil
 	subscription.Filter.ToBlock = nil
 	// We set this to the current rollup height, so that historical logs aren't returned.
-	subscription.Filter.FromBlock = s.storage.FetchHeadRollup().Number()
+	subscription.Filter.FromBlock = big.NewInt(0).Add(s.storage.FetchHeadRollup().Number(), big.NewInt(1))
```

Comment:
big int API is so weird, just can't get used to it

Reply:
Yeah, but I don't want to rewrite the big.Int API as part of this PR :(